src/ports/postgres/modules/graph/test/pagerank.sql_in

```diff
@@ -66,7 +66,12 @@ SELECT pagerank(
              'id',            -- Vertix id column
              '"EDGE"',          -- "EDGE" table
              'src=src, dest=dest', -- "EDGE" args
-             'pagerank_out'); -- Output table of PageRank
+             'pagerank_out',
+              NULL,
+              NULL,
+              NULL,
+              NULL,
+             NULL); -- Output table of PageRank
```

Comment:
Move comment '-- Output table of PageRank' to upper column.

Reply:
This is corrected as well.